### ISSUE TITLE
[no-duplicate-load-rules] False positive when using @use and @forward for the same stylesheet

### ISSUE THREAD

**Issue by reduckted** [stylelint-scss/stylelint-scss#1121]
Given this stylesheet:
```scss
@use "foo";
@use "bar";

@forward "foo"; // <-- Unexpected duplicate load rule breakpoints (scss/no-duplicate-load-rules)

.test {
    margin: foo.$margin;
}
```

The `@forward` is reported as a duplicate when it shouldn't be.

From https://sass-lang.com/documentation/at-rules/forward/ (emphasis is mine):

> The rule is written `@forward` "<url>". It loads the module at the given URL just like `@use`, but it makes the [public](https://sass-lang.com/documentation/at-rules/use#private-members) members of the loaded module available to users of your module as though they were defined directly in your module. **Those members aren’t available in your module, though—if you want that, you’ll need to write a `@use` rule as well.** Don’t worry, it’ll only load the module once!
